components.yaml

```diff
@@ -35,6 +35,12 @@ components:
     branch: master
     update-policy: static
     metadata: ""
+  multusV3:
```

Comment:
missing bump script imo
it will fail the auto bumper isnt it ?

Reply:
> missing bump script imo (even blank if should do nothing)
> it will fail the auto bumper isnt it ?

What do you mean by that?

> also which one should be the default for now by CNAO / HCO , not the v3 ?

The default remains  Multus v4.

Reply:
> > missing bump script imo (even blank if should do nothing)
> > it will fail the auto bumper isnt it ?
> 
> What do you mean by that?

Each component has a script
`./hack/components/bump-<COMPONENET_NAME>.sh`
the auto bumper is a script that consumes the yaml and runs the scripts,
if a script is missing it will fail

I would suggest to look on other component adding and check if more stuff is missing